tests-scan

```diff
@@ -142,6 +142,7 @@ def tests_invoke(priority, name, number, revision, ref, context, base, original_
 
     checkout = "PRIORITY={priority:04d} ./make-checkout --verbose"
     cmd = "TEST_PROJECT={repo} TEST_NAME={name}-{current} TEST_REVISION={revision} ../tests-invoke --pull-number={pull_number} "
+    wrapper = "TEST_PROJECT={repo} BASE_PROJECT={github_base} TEST_NAME={name}-{current} TEST_REVISION={revision} ./publish-wrapper "
```

Comment:
Here is redundancy. I thought I could just drop those env variables that `cmd` and `wrapper` share - but then I would need to reconstruct it in the wrapper in order to log the command which is being executed.